src/lib/LastManStandingUsage.ts

```diff
@@ -0,0 +1,246 @@
+export default class LastManStandingUsage {
+	public contestants = 0;
+	public deaths: Set<number> = new Set();
+	public killers: Set<number> = new Set();
+	private readonly parts: (string | number)[] = [];
+
+	public constructor(usage: string) {
+		this.parse(usage);
+	}
+
+	public display(...values: string[]) {
+		return this.parts
+			.map(part => (typeof part === 'number' ? `**${values[part]}**` : part))
+			.join('');
+	}
+
+	public parse(usage: string): void {
+		let current = '';
+		for (let i = 0, char; i < usage.length; i++) {
+			char = usage.charAt(i);
+			if (char === '{') {
+				// If there was text, push buffer
+				if (current) {
+					this.parts.push(current);
+					current = '';
+				}
+
+				// Parse tag {NT?}
+				const n = Number(usage.charAt(++i));
+
+				// If N > contestants, assign contestants to N
+				if (n > this.contestants) this.contestants = n;
+
+				const nextChar = ++i;
+				// If D, add death; if K, add killer
+				if (usage.charAt(nextChar) === 'D') {
+					this.deaths.add(n - 1);
+					i++;
+				} else if (usage.charAt(nextChar) === 'K') {
+					this.killers.add(n - 1);
+					i++;
+				}
+
+				this.parts.push(n - 1);
+			} else {
+				current += char;
+			}
+		}
+
+		if (current) this.parts.push(current);
+	}
+}
+
+export const LMS_PREP = [
+	'{1} grabs a DDS.',
+	'{1} grabs an Abyssal whip and retreats.',
+	'{1} and {2} fight for a whip. {1} gives up and retreats.',
+	'{1} and {2} fight for a whip. {2} gives up and retreats.',
+	"{1} finds a Heavy ballista, some Javelins, and an Ava's assembler.",
+	'{1} runs into upstairs in a building and hides.',
+	'{1} takes a handful of Morrigan Javelins.',
+	"{1} rips a Zuriel's staff out of {2}'s hands.",
+	'{1} finds a vial full of Saradomin brew.',
+	'{1} gathers as much food as they can.',
+	'{1} grabs a Vesta longsword.',
+	'{1} takes a Ghrazi Rapier from inside the Trinity Outpost.',
+	'{1} finds a Crate with an Infernal cape.',
+	'{1} takes a AGS from inside the Moser Settlement.',
+	'{1}, {2}, and {3} work together to get as many supplies as possible.',
+	'{1} runs away with a pair of Eternal boots and some sharks.',
+	'{1} snatches a vial of Super restore and a Stamina potion.',
+	'{1} finds a supply box full of Granite mauls & supplies.',
+	'{1} searches a supply box, not realizing it is empty.',
+	'{1}, {2}, {3}, and {4} share everything they gathered before running.',
+	'{1} retrieves a Staff of the Dead from the Debtor hideout .',
+	"{1} grabs a Stamina potion while {2} gets Ahrim's robes.",
+	'{1} scares {2} away from the Moser Settlement.',
+	'{1} grabs a Dragonfire shield from the ground on The Mountain.',
+	'{1} snatches a pair of Dragon claws.',
+	'{1} grabs a lone pair of pants.',
+	'{1} trips over while running to a building, {2} picks them up, they run off together.'
```

Comment:
'{1} trips over while running to a building, {2} picks them up and they run off together.'